tikv/kv.go

```diff
@@ -281,27 +279,33 @@ func (s *KVStore) runSafePointChecker() {
 
 // Begin a global transaction.
 func (s *KVStore) Begin() (*transaction.KVTxn, error) {
-	return s.BeginWithOption(DefaultStartTSOption())
+	return s.BeginWithOption()
 }
 
 // BeginWithOption begins a transaction with the given StartTSOption
-func (s *KVStore) BeginWithOption(options StartTSOption) (*transaction.KVTxn, error) {
-	if options.TxnScope == "" {
-		options.TxnScope = oracle.GlobalTxnScope
+func (s *KVStore) BeginWithOption(options ...StartTSOption) (*transaction.KVTxn, error) {
```

Comment:
For flexibility and extensibility, maybe use more generic name like `BeginOption`. By the way, since the arguments are optional, it seems that we can merge it with `Begin` function.
Also, for upgrading TiDB smoothly, we can first add options to `Begin` function and keep old `BeginWithOption` unchanged. After TiDB upgrades successfully, we can then remove the old method.